src/sound/audio_stream/msg/AudioData.msg

```diff
@@ -0,0 +1 @@
+uint8[] data
```

Comment:
Can't we use a standard message, instead of this?
http://wiki.ros.org/std_msgs
http://docs.ros.org/api/std_msgs/html/msg/ByteMultiArray.html
http://docs.ros.org/api/std_msgs/html/msg/UInt8MultiArray.html